exercise-1/style.css

```diff
@@ -0,0 +1,149 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  text-decoration: none;
+  list-style: none;
+}
+
+html,
+body {
+  height: 100vh;
+  margin: 0;
+  font-family: 'Lato', 'san-serif';
+}
+
+button {
+  display: none;
+}
+
+
+nav {
+  min-width: 150px;
+  display: flex;
+  justify-content: end;
+  height: 100vh;
+  color: #fff;
+  text-align: center;
+  background-color: #b2a68d;
+}
+
+nav a {
+  font-size: 16px;
+  color: #fff;
+  display: block;
+  padding: 12px;
+  padding-left: 30px;
+  transition: background-color 0.3s ease-in-out;
+  border-bottom: 1px solid;
+  -webkit-tap-highlight-color: transparent;
+}
+
+nav a:hover {
+  color: #a88c7d;
+  background: #fff;
+  outline: none;
+  position: relative;
+  background-color: #fff;
+  border-top-left-radius: 20px;
+  border-bottom-left-radius: 20px;
+}
+
+ul {
+  display: flex;
+  flex-direction: column;
+  gap: 30px;
+}
+
+ul:first-child {
+  padding-top: 30px;
+}
+
+main {
+  flex-grow: 1;
+  margin-left: 20px;
+  margin-top: 30px;
+}
+
+.container {
```

Comment:
It's a very generic class name, maybe go with `page-container` or something more specific. 🤷‍♀️